src/NzbDrone.Core/Notifications/Plex/PlexTv/PlexTvService.cs

```diff
@@ -79,5 +81,29 @@ public string GetAuthToken(int pinId)
 
             return authToken;
         }
+
+        public HttpRequest GetWatchlist(string authToken)
```

Comment:
There really wasn't a great place to put this without writing a bunch of boilerplate code to wrap plex media providers.